app/controllers/api/v1/collaborators_controller.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Api::V1::CollaboratorsController < Api::V1::BaseController
+  before_action :authenticate_user!
+  before_action :set_project
+  before_action :check_author_access, only: %i[create destroy]
+  before_action :set_collaborator, only: %i[destroy]
+
+  # /api/v1/projects/:project_id/collaborators
+  def index
+    @collaborators = paginate(@project.collaborators)
```

Comment:
Why we are exposing this endpoint to all, shoudn't this be restricted based on access to project?

Reply:
I thought we can show the collaborators to any authenticated user, does that make sense ? @tachyons

Reply:
But that is public project alone, right? cc @satu0king

Reply:
Yeah, i missed that part.. will add a check for this.. @tachyons

Reply:
@tachyons made the required changes please check..